portsyncd/portsyncd.cpp

```diff
@@ -86,13 +86,7 @@ int main(int argc, char **argv)
         netlink.dumpRequest(RTM_GETLINK);
         cout << "Listen to link messages..." << endl;
 
-        if (!handlePortConfigFromConfigDB(p, cfgDb, warm))
-        {
-            // if port config is missing in ConfigDB
-            // program will exit with failure
-            SWSS_LOG_THROW("ConfigDB does not have port information, exiting...");
-            return EXIT_FAILURE;
-        }
+        handlePortConfigFromConfigDB(p, cfgDb, warm);
```

Comment:
please have all the code as before and log error here instead of throw. Also give the comment why continuing the flow.

Reply:
@prsunny 
Sure will do. Since zero ports is becoming a valid flow, may I use log warning instead of error?

Reply:
ok

Reply:
@prsunny I gave it some more thought. 
Do we really want to add a warning here? 
Since there is no problem and we just want to notify the user that there are no ports, we might as well change it to notice. 
What do you think?

Reply:
fine with me.